### PR TITLE
Scrape stats

### PR DESCRIPTION
**Pull request by ianmah**
TODO: Run /scrape on a cron
image

### CODE REVIEW

File: backend/server.js [hunk 1]
```diff
@@ -73,4 +121,6 @@ app.get('/me', (req, res) => {
       console.log('Something went wrong!', err);
       res.send(err);
     });
-});
\ No newline at end of file
+});
+
+// cursors: { after: '1626508570260', before: '1626506057589' },
```
Comment (vandyliu) [inline backend/server.js:126]: what's this for?

File: backend/server.js [hunk 2]
```diff
@@ -31,14 +33,60 @@ app.post('/user', UserController.createUser);
...
+        data.body.items.forEach(item => {
+          const id = `listenStats.${item.track.id}`
+          if(ids[id]) {
+            ids[id] = ids[id] + 1;
+          } else {
+            ids[id] = 1
+          }
+        })
+        console.log(ids)
+        const newLastScraped = data.body.cursors?.after || lastScraped
```
Comment (vandyliu) [inline backend/server.js:87]:
```suggestion
        const newLastScraped = data.body.cursors?.after || lastScraped || 0
```
Right now, I get `lastScraped = undefined` and newLastScraped is also undefined

File: backend/server.js (hunk 1 above)
Comment (ianmah) [reply backend/server.js]: oops

**Review by vandyliu — APPROVED** [review]
looks good

**Pull request merged**